frontend/vue/components/CodeExercise/CodeEditorTools.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="code-editors-tools">
+    <div v-if="resetEnabled" class="code-editors-tools__button">
+      <bx-tooltip-icon
+        class="code-editors-tools__button__tooltip"
+        aligment="center"
+        direction="top"
+        body-text="Reset cell to initial content"
+        @click="reset"
+      >
+        <ResetIcon class="code-editors-tools__button__icon" />
+      </bx-tooltip-icon>
+    </div>
+    <div v-if="notebookEnabled" class="code-editors-tools__button">
+      <bx-tooltip-icon
+        class="code-editors-tools__button__tooltip"
+        aligment="center"
+        direction="top"
+        body-text="Copy to notebook notes"
```

Comment:
Similar comment regarding translations, something like:

```
body-text="$translate('Copy to notebook notes')"
```

then make sure the strings are defined in `strings.yaml` file